galasa-ui/src/tests/header.test.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import HomePage from '@/app/page';
+import PageHeader from '@/components/PageHeader';
+import { act, render, screen } from '@testing-library/react';
+
+jest.mock('next/headers', () => ({
+  cookies: jest.fn(() => ({
+    get: jest.fn().mockReturnValue('')
+  }))
+}))
```

Comment:
Don't think we need to mock the cookies API here since we're not using it in the header - this can be removed :)